resources/select.html.jinja

```diff
@@ -0,0 +1,48 @@
+<div style="margin-top: 0.5rem; width: 100%">
+  <label for="version-select">Version:</label>
+  <select id="version-select" name="version-select" style="text-align: center; width: 100%">
+    <option value="{{ version }}" selected="selected">{{ version }}</option>
+  </select>
+</div>
+
+<script type="text/javascript">
+  const select = document.querySelector("#version-select");
+
+  select.addEventListener("change", (e) => {
+    const selected = select.value;
+
+    // Replace the existing version number in the URL with the newly
+    // selected version:
+    let href = window.location.href;
+    href = href.replace(/[\d]+\.[\d]+\.[\d]+[^\/]*/g, selected);
```

Comment:
Just had a small shower thought, I don't think this is going to catch `1.0.0-beta.0` or `1.0.0-rc.0` etc?

Reply:
Note to self, why guess when I can just try it :D - and yes it works. Sorry for the noise :sweat_smile: